### ISSUE TITLE
Unable to skip rules inside a block

### ISSUE THREAD

**Issue by MAS150MD200** [ansible/ansible-lint#484]
# Issue Type
- Bug report

# Ansible and Ansible Lint details
```
ansible --version
ansible 2.5.5
  config file = /Users/atsvetkov/.ansible.cfg
  configured module search path = [u'/Users/atsvetkov/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Library/Python/2.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.10 (default, Oct  6 2017, 22:29:07) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.31)]

ansible-lint --version
ansible-lint 4.1.0
```

# Desired Behaviour
```
  - name: some block name
    block:
      - name: block_1
        shell: yum install vim  # noqa 303
```
ansilbe lint should ignore `yum install ...` warning.

# Actual Behaviour (Bug report only)
ansible lint produces the following output:
`[303] yum used in place of yum module`

But, ` # noqa 303` option will work if we put the same task outside of a block.

**Comment by JohnVillalovos** [comment]
I am also seeing this issue. I can work around it by using the tags -> skip_ansible_lint but that is kind of a hammer.

```
tags:
  - skip_ansible_lint
```

**Comment by madonius** [comment]
This seems to be true for `always:` blocks too.

**Comment by westurner** [comment]
I'm having the same issue with ansible 2.7.9

**Comment by westurner** [comment]
This also works but shouldn't be necessary:
```yaml
tags: [skip_ansible_lint]
```
